server/app.py

```
@@ -0,0 +1,297 @@
+            "id": node.id,
+            "role": node.role,
+            "content": node.content,
+            "branch_name": node.branch_name,
+            "children": [serialize_node(c) for c in node.children],
+            "is_current": node == tree.current_node
+        }
+    
+    return {
+        "root": serialize_node(tree.root),
+        "current_node_id": tree.current_node.id,
+        "conversation_id": target_id
```

Ruff (0.14.10)

Undefined name `target_id`

(F821)